src/ray/util/logging.cc

```diff
@@ -67,33 +58,21 @@ long RayLog::log_rotation_file_num_ = 10;
 bool RayLog::is_failure_signal_handler_installed_ = false;
 
 std::string GetCallTrace() {
-  std::string return_message = "Cannot get callstack information.";
-#if defined(RAY_USE_GLOG) || defined(RAY_USE_SPDLOG)
-  return google::GetStackTraceToString();
-#endif
-  return return_message;
-}
-
-#ifdef RAY_USE_GLOG
-struct StdoutLogger : public google::base::Logger {
-  std::ostream &out() { return std::cout; }
-
-  virtual void Write(bool /* should flush */, time_t /* timestamp */, const char *message,
-                     int length) {
-    // note: always flush otherwise it never shows up in raylet.out
-    out().write(message, length) << std::flush;
+  std::vector<void *> local_stack;
+  local_stack.resize(50);
+  absl::GetStackTrace(local_stack.data(), 50, 0);
+  static constexpr size_t buf_size = 16 * 1024;
+  char buf[buf_size];
+  std::string output;
+  for (auto &stack : local_stack) {
+    if (absl::Symbolize(stack, buf, buf_size)) {
+      output.append("    ").append(buf).append("\n");
```

Comment:
Hey is there any good way to add line information using addr2line here? https://linux.die.net/man/1/addr2line

Reply:
I am just curious if that's possible since it looks like we can access each stack trace